static/js/people.js

```diff
@@ -214,14 +214,15 @@ exports.get_recipient_count = function (person) {
         return person.pm_recipient_count;
     }
 
-    var count = pm_recipient_count_dict.get(person.email);
+    var user_id = person.user_id || person.id;
```

Comment:
Oh, I guess it's probably because messages have `id` as the variable name still?  I guess we should open an issue to do the slow migration by putting `user_id` in there too so we can eventually make it consistent in our API.